docs/realm.js

```diff
@@ -106,9 +106,11 @@ class Realm {
  * Get the current schema version of the Realm at the given path.
  * @param {string} path - The path to the file where the
  *   Realm database is stored.
+ * @param {string} encryptionKey - Optional encryption key.
```

Comment:
And perhaps the comment should instead say that it's required if the Realm is encrypted.